multiqc/modules/preseq/preseq.py

```diff
@@ -70,23 +74,114 @@ def parse_preseq_logs(self, f):
+            data = {k / self.counts_in_1x: v / self.counts_in_1x for k, v in data.items()}
+            self.axis_label = 'Coverage'
         return data
+    
+    
+    def _read_real_counts(self):
+        real_counts_file_raw = None
+        real_counts_file_name = None
+        for f in self.find_log_files('preseq/real_counts'):
+            if real_counts_file_raw is not None:
+                log.warn("Multiple Preseq real counts files found, now using {}".format(f['fn']))
+            real_counts_file_raw = f['f']
+            real_counts_file_name = f['fn']
+
+        real_counts_total = {}
+        real_counts_unique = {}
+        if real_counts_file_raw is not None:
+            try:
+                for line in real_counts_file_raw.splitlines():
+                    if not line.startswith('#'):
+                        line = line.strip()
+                        cols = line.split()  # Split on any whitespace
+                        for sn in [cols[0], "{}.preseq".format(cols[0]), "{}.bam".format(cols[0])]:
+                            if sn in self.preseq_data:
+                                if len(cols) >= 2:
+                                    if cols[1].isdigit():
+                                        real_counts_total[sn] = int(cols[1])
+                                if len(cols) >= 3:
+                                    if cols[2].isdigit():
+                                        real_counts_unique[sn] = int(cols[2])
+            except IOError as e:
+                log.error("Error loading real counts file {}: {}".format(real_counts_file_name, str(e)))
+            else:
+                log.debug("Found {} matching sets of counts from {}".format(len(real_counts_total), real_counts_file_name))
+
+        # Convert real counts to coverage
+        if self.counts_in_1x is not None:
+            for f, c in real_counts_total.items():
+                real_counts_total[f] = float(c) / self.counts_in_1x
+            for f, c in real_counts_unique.items():
+                real_counts_unique[f] = float(c) / self.counts_in_1x
+        return real_counts_total, real_counts_unique
+    
+    
+    def _real_counts_to_plot_series(self, real_counts_unique, real_counts_total):
+        series = []
+        if real_counts_total:
+            # Same defaults as HighCharts for consistency
+            default_colors = ['#7cb5ec', '#434348', '#90ed7d', '#f7a35c', '#8085e9',
+                              '#f15c80', '#e4d354', '#2b908f', '#f45b5b', '#91e8e1']
+            for si, sn in enumerate(sorted(self.preseq_data.keys())):
+                if sn in real_counts_total:
+                    t_reads = float(real_counts_total[sn])
+                    point = {
+                        'color': default_colors[si],
+                        'showInLegend': False,
+                        'marker': {
+                            'enabled': True,
+                            'symbol': 'diamond'
```

Comment:
A black border looks the most visible, will keep it like this:
![image](https://cloud.githubusercontent.com/assets/1575412/26127688/ade9c7fe-3a92-11e7-8aea-04613269d528.png)